server/config/development/cdap.json

```diff
@@ -23,5 +23,6 @@
   "hsts.enabled": "false",
   "hsts.max.age": 31536000,
   "hsts.include.sub.domains": "true",
-  "hsts.preload": "true"
+  "hsts.preload": "true",
+  "app.cdap.file.max.size.bytes": "2097152"
```

Comment:
why do we have the default value here if it is now coming from backend?

This is an anti-pattern; default values should be configured in a single place, either in the UI or the backend, to ensure consistency and maintainability.

Reply:
This is not a default value, these are just for development.